dev/reference/tools/sync-diff-inspector/tidb-tidb-diff.md

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
Why two `tidb` in the filename? `tidb-tidb-diff.md`

Reply:
means compare data between two tidb cluster.